### DISCUSSION TITLE
How to use new or tabnew in coc-definition jump?

### DISCUSSION THREAD

**Discussion by q962** [neoclide/coc.nvim#4369] [Q&A] [answered]
Prefer new or tabnew when the document is not saved

Can it be modified through configuration?
Of course, it also includes other jump commands.

**Comment by fannheyward** [comment, marked as answer]
Set `coc.preferences.jumpCommand` in your coc-settings.json, check `:h coc-preferences-jumpCommand`.

**Comment by chemzqm** [comment]
You can use `CocAction('jumpDefinition')`
